Main.java

```diff
@@ -24,8 +24,8 @@ public static float getMean(int[] arr) {
     public static float getMedian(int[] arr) {
     	Arrays.sort(arr);
     	if(arr.length%2 == 0) {//check if there are even amount of numbers
-			int mid = 1 + (int)(arr.length/2);
-			return (arr[mid]+arr[mid+1])/2;
+			int mid = arr.length/2;
+			return (arr[mid]+arr[mid-1])/2;
 			}
 		else {
 			int mid = (int)(arr.length/2);
```

Comment:
These still need a formatting pass to conform with our standards - you can use Alt+Shift+F in VS Code or follow the format of the existing methods to do this.